### ISSUE TITLE
Drawer: filter apps already added to other tabs or folders

### ISSUE THREAD

**Issue by m3thm4th** [otakuhqz/Omega#420]
When categorizing apps into tabs or folders, creating a new tab/folder and tapping on "Manage apps", the three dots menu only has the voice "Reset Selection". A switch could be added, that would filter out (hide) apps that have already been added to other tabs or folders, depending on the case, so users can avoid adding them to two categories (if they so wish).

**Comment by jterrain** [comment]
Agreed. Importantly, this would also save significant time scrolling down a long list of apps repeatedly that are already added to other tabs and folders.
